### DISCUSSION TITLE
A few simple questions about GitHub Desktop

### DISCUSSION THREAD

**Discussion by TristanTheLamb** [community/community#33512] [Other Feature Feedback, Questions, & Ideas] [answered]
What is GitHub Desktop built on? 

Does GitHub Desktop store your customer’s data? If so where & how is the data stored?

What additional security measures does GitHub Desktop include?

does GitHub Desktop have SSO or other team/enterprise features? 

Where can I find a roadmap that shows features GitHub Desktop will add in the future?

**Comment by DenverCoder1** [comment, marked as answer]
Many of these questions can be answered by this GitHub repository:

https://github.com/desktop/desktop
